torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2574,6 +2574,20 @@ def sample_inputs_trace(self, device, dtype, requires_grad, **kwargs):
                                      low=None, high=None,
                                      requires_grad=requires_grad))),)
 
+def sample_inputs_linalg_trace(self, device, dtype, requires_grad, **kwargs):
+    inputs = (
+        ((S, S), 0),
```

Comment:
What happens on empty tensors? We should probably add a case for them. What about a batched sample input too? `(S, S, S`

Reply:
The trace function implemented in this PR returns different values from `numpy.trace` for 3-dim inputs. `numpy.trace` reduces with axis1=0, axis2=1 whereas array API specifies to reduce with axis1=-2, axis2=-1.

Reply:
Is it possible to compare then against a lambda with our same defaults that the  calls into np.trace?